`make build-debs` doesn't explicitly ignore config.py

If you run `make securedrop/config.py && make build-debs`, it'll fail with:

```
FAILED builder/tests/test_securedrop_deb_package.py::test_app_code_paths_missing[/var/www/securedrop/config.py] - Failed: found -rw-r--r-- root/root      3724 2023-08-18 11:56 ./var/www/securedrop/config.py
```

This just needs a fix like https://github.com/freedomofpress/securedrop/commit/62b195f71cd93f951b85d855a90b3dc70007c0ce. It's not urgent because the workaround is trivial, just delete your local config.py before packaging.

--- a/securedrop/debian/rules
+++ b/securedrop/debian/rules
@@ -49,6 +49,8 @@ override_dh_install:
 	rm upload-screenshots.py
 	# If a custom logo exists in the tree, delete it
 	rm -f static/i/custom_logo.png
+	# Same with any config.py
+	rm -f config.py
 	dh_install
 
 override_dh_strip_nondeterminism:

